src/Translator.php

```diff
@@ -435,7 +435,7 @@ protected function assertValidLocale($locale)
 	 */
 	private function extractMessageDomain($message)
 	{
-		if (strpos($message, '.') !== FALSE && strpos($message, ' ') === FALSE) {
+		if (strpos($message, '.') !== FALSE && strpos(strrev($message), '.') !== 0 && strpos($message, ' ') === FALSE) {
```

Comment:
What do you think about this?

```suggestion
		if (strpos(substr($message, 0, -1), '.') !== FALSE && strpos($message, ' ') === FALSE) {
```

Reply:
it works, I'm in!